TypeCobol.Test/Misc/CodeElementDebugTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Nodes;
+using TypeCobol.Test.Utils;
+
+namespace TypeCobol.Test.Misc
+{
+    [TestClass]
+    public class CodeElementDebugTest
+    {
+        /// <summary>
+        /// Issue #1878 : check the debug mode of CodeElement
+        /// </summary>
+        [TestMethod]
+        [TestCategory("Parsing")]
+        [TestProperty("Time", "fast")]
+        public void CheckCodeElementDebugProperty()
+        {
+            var expectedDebugMode = new List<CodeElement.DebugType>()
+            {
+                CodeElement.DebugType.None, // CodeElement 0
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None, //5
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.None, //10
+                CodeElement.DebugType.Mix,
+                CodeElement.DebugType.None,
+                CodeElement.DebugType.All,
+                CodeElement.DebugType.Unset, //    .
+                CodeElement.DebugType.All,  //15
+                CodeElement.DebugType.Unset, //    .
+                CodeElement.DebugType.Mix, 
+                CodeElement.DebugType.Mix, 
+                CodeElement.DebugType.Mix,
+                CodeElement.DebugType.Mix, //20
+                CodeElement.DebugType.Mix, 
+                CodeElement.DebugType.All, 
+                CodeElement.DebugType.Unset, //    GO BACK
```

Comment:
`Unset` seems a bug here, the property should always be set.

Reply:
Indeed, it was a bug, corrected now.